docs/release-notes/8.5.asciidoc

```diff
@@ -1,6 +1,21 @@
 [[release-notes-header-8.5.0]]
 == 8.5
 
+[discrete]
+[[release-notes-8.5.1]]
+=== 8.5.1
+
+[discrete]
+[[bug-fixes-8.5.1]]
+==== Bug fixes and enhancements
+* Fixes a bug that caused {elastic-endpoint}s (version 8.5.0) running on Linux systems with many CPUs to sometimes become unhealthy https://github.com/elastic/endpoint/issues/34[#34].
+* Fixes bug that caused incorrect alerts to display in Timeline when investigating alerts from the Detection & Response dashboard ({pull}144319[#144319]).
+* Updates the User authentication area chart so it can be opened in Lens ({pull}144011[#144011]).
+* Fixes the {jira} connector icon for users with a basic license ({pull}143916[#143916]).
+* Updates the link in the machine learning rule type card to direct users towards the Elastic licensing page ({pull}143836[#143836]).
+* Turns off the option to edit machine learning rules if users don’t have have the appropriate Machine Learning privileges ({pull}143260[#143260]).
+* Removes the ability from the UI to enable and disable machine learning rules if users don’t have the appropriate Machine Learning permissions({pull}143252[#143252]).
```

Comment:
If this is referring to the specific Kibana privilege labeled "Machine Learning," then maybe capitalize but in that case it should also probably be something like "if users don't have the Machine Learning privilege in Kibana." But it sounds like this might be intended as a more general catch-all, so that's how I edited it (also added a missing space):
```suggestion
* Turns off the option to edit machine learning rules if users don’t have have the appropriate machine learning privileges ({pull}143260[#143260]).
* Removes the ability from the UI to enable and disable machine learning rules if users don’t have the appropriate machine learning privileges ({pull}143252[#143252]).
```

Reply:
@maximpn just want to verify one more time that these are accurate:

```suggestion
* Turns off the option to edit machine learning rules if users don’t have have Machine Learning privileges in Kibana set to `All` ({pull}143260[#143260]).
* Removes the ability from the UI to enable and disable machine learning rules if users don’t have have Machine Learning privileges in Kibana set to `All` ({pull}143252[#143252]).
```

Reply:
@nastasha-solomon  LGTM